frontend/src/metabase/lib/card.js

```diff
@@ -56,7 +56,7 @@ export function isCardDirty(card, originalCard) {
         }
     } else {
         const origCardSerialized = originalCard ? serializeCardForUrl(originalCard) : null;
-        const newCardSerialized = card ? serializeCardForUrl(card) : null;
+        const newCardSerialized = card ? serializeCardForUrl(_.omit(card, 'original_card_id')) : null;
```

Comment:
Can we just have `serializeCardForUrl` not include `original_card_id`, or does it need to be included in some cases?